README.md

```diff
@@ -1,5 +1,7 @@
 # stac-mjx :rat:
-Implementation of [STAC](https://ieeexplore.ieee.org/document/7030016) using [MJX](https://mujoco.readthedocs.io/en/stable/mjx.html). This is part of the VNL project. 
+`stac-mjx` is an implementation of the [Stac](https://ieeexplore.ieee.org/document/7030016) algorithm for inverse kinematics on markerless motion capture data. It's written in [MJX](https://mujoco.readthedocs.io/en/stable/mjx.html) for hardware acceleration . 
```

Comment:
"It uses [JAX](https://jax.readthedocs.io/en/latest/) with [Mujoco XLA (MJX)](https://mujoco.readthedocs.io/en/stable/mjx.html) for GPU acceleration."